packages/skatejs/src/with-children.js

```diff
@@ -19,7 +19,7 @@ export const withChildren = (Base: Class<any> = HTMLElement): Class<any> =>
         const fn = this.childrenUpdated.bind(this);
         this._mo = new MutationObserver(fn);
         this._mo.observe(this, { childList: true });
-        fn();
+        documentReady().then(fn);
```

Comment:
Circling back to the discussion, I'm happy for this to go in. Looks like this could just be `document.addEventListener('DOMContentLoaded', fn)` though.

Reply:
Cool, cheers. Might be worth porting that to the v6-trey branch, too.